src/CaptureClient/CaptureClient.cpp

```diff
@@ -41,7 +40,7 @@ using orbit_grpc_protos::CaptureResponse;
 using orbit_grpc_protos::ClientCaptureEvent;
 using orbit_grpc_protos::InstrumentedFunction;
 using orbit_grpc_protos::TracepointInfo;
-using orbit_grpc_protos::UnwindingMethod;
+using UnwindingMethod = orbit_grpc_protos::CaptureOptions::UnwindingMethod;
```

Comment:
I'm not too sure I like this alias instead of the full `CaptureOptions::UnwindingMethod` across these files. The reason is that the possible values are then still in `CaptureOptions` (e.g., `CaptureOptions::kFramePointers`) so it looks a bit weird to refer to the possible values with the prefix but to the type without.

Reply:
Hmm, it is that versus more verbose type names. I leave it as is.